tickets/src/utils/links.ts

```diff
@@ -28,8 +28,8 @@ export function googleCalendarLinkBuilder(
     )
   } else {
     const endDate = new Date(date)
-    endDate.setDate(date.getDate() + 1)
-    end = formatDateForCalendar(endDate, true)
+    endDate.setHours(date.getHours() + 1) // Set events to last an hour by default
```

Comment:
Would it make sense to add a duration setting to the event creation form? If wo do you mind opening an issue so we can keep track of this? Thanks!